jsonnet/templates/ocp-performance.jsonnet

```
@@ -283,9 +388,65 @@ local crioMemory = grafana.graphPanel.new(
 
 // Cluster details
 
+local current_node_count = grafana.statPanel.new(
+  title='Current Node Count',
+  datasource='$datasource',
+).addTarget(
+  prometheus.target(
+    'sum(kube_node_info{clustername=~"$clustername"})',
+    legendFormat='Number of nodes',
+  )
+).addTarget(
+  prometheus.target(
+    'sum(kube_node_status_condition{status="true"}) by (condition)',
+    legendFormat='Node: {{ condition }}',
+  )
+);
+
+local current_namespace_count = grafana.statPanel.new(
+  title='Current Namespace Count',
+  datasource='$datasource',
+  graphMode='none',
+).addTarget(
+  prometheus.target(
+    'count(kube_namespace_created{clustername=~"$clustername"})',
+    legendFormat='Namespace Count',
+  )
+);
+
+local current_pod_count = grafana.statPanel.new(
+  title='Current Pod Count',
+  datasource='$datasource',
+).addTarget(
+  prometheus.target(
+    'sum(kube_pod_status_phase{phase="Running",namespace=~"$namespace",clustername=~"$clustername"})',
+    legendFormat='Running Pods',
+  )
+).addTarget(
+  prometheus.target(
+    'sum(kube_pod_status_phase{phase="Succeeded",namespace=~"$namespace",clustername=~"$clustername"})',
+    legendFormat='Succeeded Pods',
+  )
+).addTarget(
+  prometheus.target(
+    'sum(kube_pod_status_phase{phase="Failed",namespace=~"$namespace",clustername=~"$clustername"})',
+    legendFormat='Failed Pods',
+  )
+).addTarget(
+  prometheus.target(
+    'sum(kube_pod_status_phase{phase="Failed",namespace=~"$namespace",clustername=~"$clustername"})',
+    legendFormat='Failed Pods',
+  )
+);
```

Can be implemented in a single query:
```suggestion
.addTarget(
  prometheus.target(
    'sum(kube_pod_status_phase{namespace=~"$namespace",clustername=~"$clustername"}) by (phase)',
    legendFormat='{{ phase}} Pods',
  )
).
```